### PR TITLE
Migrate to gatsby v2 and fix other issues

### PR DESCRIPTION
**Pull request by sidharthachatterjee**

### CODE REVIEW

File: src/html.js [hunk 1]
```diff
@@ -1,26 +1,8 @@
 import React from "react"
```
Comment (m-allanson) [inline src/html.js]: This whole file can be deleted now, I think?

File: src/components/Section/One.js [hunk 2]
```diff
@@ -202,7 +202,7 @@ const DivInLi = styled.div `
     }
 `
 
-const SubtitleLi = styled.li `
+const SubtitleLi = styled.div `
```
Comment (m-allanson) [inline src/components/Section/One.js]: This `li` element was nested inside other elements without a parent `ul` or `ol` element, which is invalid html and was causing some very strange rendering issues in different browsers.

File: package.json [hunk 3]
```diff
@@ -12,19 +12,18 @@
     "animejs": "^2.2.0",
     "babel-plugin-styled-components": "^1.9.2",
     "dateformat": "^3.0.3",
-    "gatsby": "^1.9.277",
-    "gatsby-link": "^1.6.46",
+    "gatsby": "^2.0.64",
```
Comment (m-allanson) [inline package.json]: Some of the Gatsby plugin versions were for Gatsby v2, but Gatsby v1 was installed which was causing some of the issues you saw.

File: src/html.js (hunk 1 above)
Comment (sidharthachatterjee) [reply src/html.js]: There were some meta tags that were already added in head so I didn't want to undo that 

@sashaboulouds, you might want to use https://www.gatsbyjs.org/starters?d=gatsby-plugin-react-helmet for this later

**Pull request closed without merging**